__tests__/app.test.js

```diff
@@ -115,25 +115,40 @@ describe('GET /api/reviews', () => {
                     votes: expect.any(Number),
                     category: expect.any(String),
                     owner: expect.any(String),
-                    created_at: expect.any(String)
+                    created_at: expect.any(String),
                 })
             })
         });
         test('responds with a the specifc review object requested', () => {
-            return request(app).get('/api/reviews/4').expect(200).then((response) => {
+            return request(app).get('/api/reviews/2').expect(200).then((response) => {
                 expect(response.body).toMatchObject({ review: {
-                    review_id: 4,
-                    title: 'Dolor reprehenderit',
-                    designer: 'Gamey McGameface',
-                    owner: 'mallionaire',
-                    review_img_url: 'https://images.pexels.com/photos/278918/pexels-photo-278918.jpeg?w=700&h=700',
-                    review_body: 'Consequat velit occaecat voluptate do. Dolor pariatur fugiat sint et proident ex do consequat est. Nisi minim laboris mollit cupidatat et adipisicing laborum do. Sint sit tempor officia pariatur duis ullamco labore ipsum nisi voluptate nulla eu veniam. Et do ad id dolore id cillum non non culpa. Cillum mollit dolor dolore excepteur aliquip. Cillum aliquip quis aute enim anim ex laborum officia. Aliqua magna elit reprehenderit Lorem elit non laboris irure qui aliquip ad proident. Qui enim mollit Lorem labore eiusmod',
-                    category: 'social deduction',
-                    created_at: "2021-01-22T11:35:50.936Z",
-                    votes: 7
+                    review_id: 2,
+                    title: 'Jenga',
+                    category: 'dexterity',
+                    designer: 'Leslie Scott',
+                    owner: 'philippaclaire9',
+                    review_body: 'Fiddly fun for all the family',
+                    review_img_url: 'https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700',
+                    created_at: '2021-01-18T10:01:41.251Z',
+                    votes: 5,
                 }})
             })
         });
+        test('responds with a specifc object and includes a comment count', () => {
+            return request(app).get('/api/reviews/2').expect(200).then((response) => {
+                expect(response.body.review).toMatchObject({
+                    review_id : 2,
+                    title: expect.any(String),
+                    review_body: expect.any(String),
+                    designer: expect.any(String),
+                    votes: expect.any(Number),
+                    category: expect.any(String),
+                    owner: expect.any(String),
+                    created_at: expect.any(String),
```

Comment:
the comment_count is the only thing necessary to test here. The test above has proven that all the other properties are present